### DISCUSSION TITLE
How to change the category banner size?

### DISCUSSION THREAD

**Discussion by sudeep2003** [bagisto/bagisto#10170] [Q&A]
The category banner size is 1320 x 300 px. But I need a big size for my website. So what can I do?

**Comment by amit-webkul** [comment]
Here’s the corrected version:

---

Hi @sudeep2003,

If you want to change the banner size, you'll need to make some changes on your end of the code. For the admin side, navigate to the file paths mentioned below and modify the predefined sizes.

**Admin File Path 1:**  
https://github.com/bagisto/bagisto/blob/9b9a32a0aedf1b8d56f587e36653c835f8163c22/packages/Webkul/Admin/src/Resources/views/catalog/categories/create.blade.php#L163

**Admin File Path 2:**  
https://github.com/bagisto/bagisto/blob/9b9a32a0aedf1b8d56f587e36653c835f8163c22/packages/Webkul/Admin/src/Resources/views/catalog/categories/edit.blade.php#L211

After making the changes in the admin side, you’ll also need to update one small thing for the storefront. Please navigate to the file path mentioned below and add your desired banner size. Currently, we are using the aspect ratio as predefined.

**Shop File Path:**  
https://github.com/bagisto/bagisto/blob/9b9a32a0aedf1b8d56f587e36653c835f8163c22/packages/Webkul/Shop/src/Resources/views/categories/view.blade.php#L32
